src/prefect/runner/runner.py

```diff
@@ -94,10 +97,11 @@ class Runner:
     def __init__(
         self,
         name: Optional[str] = None,
-        query_seconds: float = 10,
+        query_seconds: float = None,
```

Comment:
Strictly speaking, I think these `parameter: type = None` should be `parameter: Optional[type] = None`

Reply:
Yea, I'm always a minimalist with that sort of thing (I believe Python type checkers do the right thing in these instances) but am definitely game to update for consistency

Reply:
udpated